common/app/redux/actions.js

```diff
@@ -1,6 +1,7 @@
 import { Observable } from 'rx';
 import { createAction } from 'redux-actions';
 import types from './types';
+import noop from 'lodash/noop';
```

Comment:
You only need to define the noop once. Otherwise you are creating the noop several times throughout the app